### ISSUE TITLE
Obelisk Placement Preventing Chalk Rune

### ISSUE THREAD

**Issue by Talryk** [Mithion/ArsMagica2#1495]
I'd noticed that when I place an obelisk and link it to a mana battery, then removing the obelisk with a pick will prevent placing of chalk runes where the obelisk used to be. For example, if I'd placed it down with just a chalk circle, then wanted to move it a single block in a direction to make room for pillars, the first spot I placed the obelisk at will not allow a rune to be placed after removing the obelisk.

This appears to happen after pairing to any destination, though I'd only checked a mana battery, calefactor, and essence refiner. Tested in SSP and Creative, but not SMP. Same issues apply to black aurem and celestial prism.

**Comment by ThatOneTraitor** [comment]
simmilar bug. with summoning the ender guardian. if you try spawning it in the same place again, it wont work, you have to break where the aurem was and place a new one.

**Comment by ThatOneTraitor** [comment]
Ill post screenshots when I can of the ender guardian related one. What im guessing is that the obelisk isnt being updated as broken?

**Comment by ThatOneTraitor** [comment]
Turns out that i have a corrupt drive.. i wont be able to take screenshots. yaay...

**Comment by TheIcyOne** [comment]
Does anything odd appear if you're wearing Magitech's Goggles when the issue occurs?

**Comment by ThatOneTraitor** [comment]
idk, mind hopping in game? ik this happens on 008. not sure about 009

**Comment by TheIcyOne** [comment]
Yeah, breaking the obelisk leaves a broken power link, and it looks like the block it creates only supports placing blocks which are blocks in item form, rather than items used to place blocks like the chalk.

**Comment by Talryk** [comment]
Found out that unpairing the destination, such as the mana battery, to which the former obelisk was linked, will allow the Broken Power Link block to be removed. Once it's gone, chalk goes down just fine.
